client/my-sites/upgrades/domain-management/components/domain/maintenance-card.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React from 'react' ;
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import { getSelectedDomain } from 'lib/domains';
+import Notice from 'components/notice';
+
+const MaintenanceCard = ( { domains, selectedDomainName, translate } ) => {
+	const domain = getSelectedDomain( { domains, selectedDomainName } ),
+		lastIndexOfDot = domain.name.lastIndexOf( '.' ),
+		tld = lastIndexOfDot !== -1 && domain.name.substring( lastIndexOfDot );
```

Comment:
True. We should probably port a file from the back-end with all the available TLDs, and check `endsWith`, and extract it to a helper. 👍 
But I'd rather do that in a new PR, and refactor this and another usage we have for it. :)